xlml/utils/xpk.py

```diff
@@ -37,98 +40,122 @@ def generate_workload_id(benchmark_id: str) -> str:
   return f"{short_benchmark}{short_id}"
 
 
-def _run_workload(
+@task
+def run_workload(
     task_id: str,
     cluster_project: str,
     zone: str,
     cluster_name: str,
     benchmark_id: str,
     workload_id: str,
+    gcs_path: str,
     docker_image: str,
     accelerator_type: str,
     run_cmds: str,
     num_slices: int = 1,
 ):
   """Run workload through xpk tool."""
-  import subprocess
+  from subprocess import run, STDOUT, PIPE
```

Comment:
Do these have to be imported inside `run_workload`? This is a plain `@task`, so I expect global imports to work.

Reply:
They could be global, but I believe @RissyRan mentioned it's better to import locally for dag parsing time?

Reply:
That makes sense for big libraries like TF, but these two imports are going to be more-or-less instant.

Reply:
I think we could have some best practices during refactors. Sometimes it's hard to tell if this dependency is big or small.